Content.Server/Chat/ChatManager.cs

```diff
@@ -112,6 +114,24 @@ public void SendDeadChat(IPlayerSession player, string message)
             _netManager.ServerSendToMany(msg, clients.ToList());
         }
 
+        public void SendAdminChat(IPlayerSession player, string message)
+        {
+            if(!_conGroupController.CanCommand(player, "asay"))
+            {
+                SendOOC(player, message);
+                return;
+            }
+            var clients = _playerManager.GetPlayersBy(x => _conGroupController.CanCommand(x, "asay")).Select(p => p.ConnectedClient);;
+
+            var msg = _netManager.CreateNetMessage<MsgChatMessage>();
+
+            msg.Channel = ChatChannel.AdminChat;
+            msg.Message = message;
+            msg.MessageWrap = $"{_localizationManager.GetString("ADMIN")}: {player.SessionId}: {{0}}";
+            msg.SenderEntity = player.AttachedEntityUid.GetValueOrDefault();
```

Comment:
Don't bother sending `SenderEntity` here, it's intended for attaching speech bubbles and such which obviously isn't applicable.

Reply:
Done.